roles/ansible-runner/tasks/runner.yml

```diff
@@ -0,0 +1,49 @@
+- name: Download source
+  git:
+    repo: "{{ item.repo }}"
+    dest: "/opt/source/{{ item.name }}"
+    update: true
+
+- name: Install config file
+  template:
+    src: ansible-runner-config
+    dest: /etc/default/{{ item.name }}
+    mode: 0600
+    owner: "{{ item.user }}"
```

Comment:
Is there a reason we are starting to work in `/opt/source/`, and then moving to `/etc/default/`, and then back to `/opt/source/`? I would suggest either moving this block up to the beginning, or after `Install playbook deps`.